str.go

```diff
@@ -254,23 +260,50 @@ func StrToRRuleSet(s string) (*Set, error) {
 // StrSliceToRRuleSet converts given str slice to RRuleSet
 func StrSliceToRRuleSet(ss []string) (*Set, error) {
 	set := Set{}
+
+	// According to RFC DTSTART is always the first line.
```

Comment:
Yes, there is no such example in the RFC, but I asked that because I can't see such thing as `RRULE set` in rfc either. There is a thing called `Recurrence set` but it is not related to the issue.
Can you please cite the exact words from RFC telling about that (maybe I am blind)?

Reply:
OK! So let it be as it is) 
I still don't like the RFC flag thing, but that is just my opinion)

Reply:
Yes, as my project continues I will likely need to make another PR, at which time I plan to resolve the comments you have made here. Thanks for your feedback!